tests/tinderbox_scraper/test_tinderbox_scraper.py

```diff
@@ -142,6 +142,22 @@
      'target': 'mozilla-central-firefox-25.0a1.en-US.linux-i686.txt',
      'target_url': 'firefox/tinderbox-builds/mozilla-central-linux/'
                    '1374583608/firefox-25.0a1.en-US.linux-i686.txt'},
+    # -a firefox -t tinderbox -p win32 --branch=mozilla-beta --extension=txt
```

Comment:
`--extension=txt` is not only valid for the `mozilla-beta` branch but for all branches. Please check the test above. Given that all other tests are based on `mozilla-central` please convert the win32 and mac case to `mozilla-central`.